src/ext/java/org/opentripplanner/ext/fares/impl/DefaultFareService.java

```diff
@@ -202,11 +269,11 @@ protected Optional<FareAndId> getBestFareAndId(
     String startZone = firstRide.getFrom().stop.getFirstZoneAsString();
     String endZone = null;
     // stops don't really have an agency id, they have the per-feed default id
-    String feedId = firstRide.getTrip().getId().getFeedId();
+    String feedId = firstRide.getAgency().getId().getFeedId(); //getTrip().getId().getFeedId();
```

Comment:
```suggestion
    String feedId = firstRide.getAgency().getId().getFeedId();
```

Reply:
Although I guess the old version where the feedId was fetched through the trip should work just as well but this works as well so just removing the comment is fine.

Reply:
Sure, will remove the comment. Just as a background for this change, the old version worked fine when testing with real data from the GTFS package, but the testing helper methods always gave the trip the same feed id regardless of the feed of the route or agency, which complicated testing. As the agency and trip should be in the same feed, it was just easier to do this small change here